sflow/agent.go

```diff
@@ -89,8 +92,12 @@ func (sfa *Agent) feedFlowTable() {
 
 		// TODO use gopacket.NoCopy ? instead of gopacket.Default
 		p := gopacket.NewPacket(buf[:n], layers.LayerTypeSFlow, gopacket.DecodeOptions{NoCopy: true})
+		logging.GetLogger().Infof("value of gopacket %s", p)
```

Comment:
It would be better to not log at all or at least to log using DEBUG otherwise in production it will generate a lot of noise

Reply:
Okay